### DISCUSSION TITLE
Google search not being executed in Agent

### DISCUSSION THREAD

**Discussion by michnovka** [danny-avila/LibreChat#4928] [Troubleshooting]
### What happened?

My Agent often does not do google search when it should.

How can I explicitly configure it to perform google search when I tell him to google something?

### Steps to Reproduce

1. configure Agent (I used OpenAI gpt-4o)
2. add google search
3. ask random questions that require google, or explicitly ask agent to google sth

### Screenshots

![image](https://github.com/user-attachments/assets/d41d976f-98a1-401f-800a-c4946c06d3aa)

![image](https://github.com/user-attachments/assets/23ca6ba9-5238-4e80-9c67-01c7a56d20cb)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You don't have Google Search properly configured, and there's not enough information from the logs to determine the issue.

![chrome_Ka5uzkMVeZ](https://github.com/user-attachments/assets/9a27a5a9-ec4c-46b5-997c-a225ac928910)

Make sure you save the agent when adding tools. More info on google search:

https://www.librechat.ai/docs/configuration/tools/google_search
